index.yaml

```diff
@@ -42,21 +42,21 @@ distributions:
     source_builds:
       default: kinetic/source-build.yaml
 #      dj: kinetic/source-jessie-build.yaml
-  lunar:
-    doc_builds:
-      default: lunar/doc-build.yaml
-      released-packages-without-doc-job: lunar/doc-released-build.yaml
-    notification_emails:
-    - ros-buildfarm-lunar@googlegroups.com
-    - clalancette+buildfarm@osrfoundation.org
-    release_builds:
-      default: lunar/release-build.yaml
-      ds: lunar/release-stretch-build.yaml
-      dsv8: lunar/release-stretch-arm64-build.yaml
-      uxhf: lunar/release-xenial-armhf-build.yaml
-      uxv8: lunar/release-xenial-arm64-build.yaml
-    source_builds:
-      default: lunar/source-build.yaml
+#  lunar:
```

Comment:
I know that it's been standard practice but what value do we gain from leaving the commented out distros in place rather than knowing that their previous configuration is preserved in Git history?

Reply:
Yeah, I don't know.  I'd be all for just removing them.  @tfoote , opinion?

Reply:
And for that matter, we can also *probably* remove the contents of the indigo, jade, and lunar subdirectories as well.

Reply:
it makes it easier to check historical settings, and or others to fork and reproduce in the future. But maybe a tag or something might make more sense for that then we can delete them all.